introduction.md

```diff
@@ -5,3 +5,8 @@ But I probably have to pick something from the list
 So it is
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?